apps/workers/bq-load/worker.go

```diff
@@ -22,12 +22,15 @@ type worker struct {
 	task.Meta
 	options
 
-	Destination `uri:"dest_table" required:"true"`
-	File        string            `uri:"origin" required:"true"`
-	Truncate    bool              `uri:"truncate"`
-	Append      bool              `uri:"append"`
-	DeleteMap   map[string]string `uri:"delete"` // will replace the data by removing current data
-	delete      bool
+	Destination `uri:"dest_table" required:"true"` // BQ table to load data into
+
+	File      string            `uri:"origin" required:"true"`   // if not GCS ref must be file, can be folder (for GCS)
+	FromGCS   bool              `uri:"from_gcs" default:"false"` // load from GCS ref, can use wildcards *
```

Comment:
Should we just check the file path and do this automatically for gs:// paths?
I like it as an option as it give more flexibility, but I don't think the option is very clear to someone that doesn't know the internals

Reply:
This thought did cross my mind too. I just didn't know if it would break backwards compatibility if we just assumed the user wanted to use the GCS reference, even if the file is in GS.  I assume it would probably work the same way as the reader in any case.